packages/cli-tools/test/storage-node.test.ts

```diff
@@ -30,4 +31,24 @@ describe('storage node', () => {
         const outputLines = await runCommand('storage-node list')
         expect(outputLines.join()).toMatch(DOCKER_DEV_STORAGE_NODE.toLowerCase())
     })
+
+    it('register storage node, show info, and finally unregister', async () => {
+        const privateKey = await createTestPrivateKey({ gas: true })
+        const publicAddress = new Wallet(privateKey).address
```

Comment:
Alternatively could be possible use:

```suggestion
        const wallet = await createTestWallet({ gas: true })
```

Reply:
Addressed in 394bb102d8243457c6bc01c8a1f76e93d7ccd737